README.md

```diff
@@ -139,12 +136,12 @@ const server = http.createServer((req, res) => {
 });
 ```
 
-## Getting an appContextId (Advanced)
-Apps using DQL or writeback features are required to supply an appContextId as part of the proxy configuration. This allows the proxy to know how to properly route requests. The appContextId can be found as part of the URL for the iframe in which your app is displayed. It will be of the form `XXXXXXXX-XXXX-4XXX-XXXX-XXXXXXXXXXXX`. To find the ID:
+## Getting a proxyId (Advanced)
+Apps using DQL or writeback features are required to supply an proxyId as part of the proxy configuration. This allows the proxy to know how to properly route requests. The proxyId can be found as part of the URL for the iframe in which your app is displayed. It will be of the form `XXXXXXXX-XXXX-4XXX-XXXX-XXXXXXXXXXXX`. To find the ID:
```

Comment:
```suggestion
Apps using DQL, writeback, or oAuth features are required to supply an proxyId as part of the proxy configuration. This allows the proxy to know how to properly route requests. The proxyId can be found as part of the URL for the iframe in which your app is displayed. It will be of the form `XXXXXXXX-XXXX-4XXX-XXXX-XXXXXXXXXXXX`. To find the ID:
```